starport/services/chain/init.go

```diff
@@ -83,6 +82,23 @@ func (c *Chain) Init(ctx context.Context) error {
 						return err
 					}
 				}
+
+				// Save a copy of the generated genesis
+				genesisData, err := ioutil.ReadFile(c.GenesisPath())
```

Comment:
I think this logic should be put [here](https://github.com/tendermint/starport/blob/develop/starport/services/networkbuilder/blockchain.go#L83) in `networkbuilder` since backing up the Genesis is a special case for `networkbuilder` service.